extensions/statuspage-monitor/package.json

```
@@ -0,0 +1,41 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "statuspage-monitor",
+  "title": "Statuspage Monitor",
+  "description": "View and monitor the status of services using Atlassian Statuspage inside Raycast! View specific component and overall status and current issues of many services on the web.",
+  "icon": "command-icon.png",
+  "author": "coding",
+  "categories": [
+    "Data",
+    "Developer Tools",
+    "Web"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "viewStatus",
+      "title": "View Status",
+      "subtitle": "Statuspage",
```

It helps with search in Raycast if the subtitle of commands is the name of the extension
```suggestion
      "subtitle": "Statuspage Monitor",
```